test/selenium/util/Driver.scala

```diff
@@ -1,18 +1,35 @@
 package selenium.util
 
+import java.net.URL
 import io.github.bonigarcia.wdm.ChromeDriverManager
 import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.remote.{DesiredCapabilities, RemoteWebDriver}
 import org.openqa.selenium.{Cookie, WebDriver}
 
 object Driver {
 
   def apply(): WebDriver = driver
 
-  private val driver: WebDriver = {
+  private lazy val driver: WebDriver =
+    if (Config.webDriverRemoteUrl.isEmpty)
+      instantiateLocalBrowser()
+    else
+      instantiateRemoteBrowser()
+
+  // Used by dev environments to run tests locally
+  private def instantiateLocalBrowser(): WebDriver = {
     ChromeDriverManager.getInstance().setup()
     new ChromeDriver()
   }
 
+  // Used by Travis to run tests in SauceLabs
+  private def instantiateRemoteBrowser(): WebDriver = {
+    val caps = DesiredCapabilities.chrome()
```

Comment:
Aha, excellent reference. I had a feeling that's why I changed them on membership, but I couldn't remember whether it was because Stripe was different or because PayPal was flaky.